speech/google/cloud/speech/sample.py

```diff
@@ -14,15 +14,19 @@
 
 """Sample class to handle content for Google Cloud Speech API."""
 
+from io import BytesIO
+
+from google.cloud._helpers import _to_bytes
+
 from google.cloud.speech.encoding import Encoding
 from google.cloud.speech.result import StreamingSpeechResult
 
 
 class Sample(object):
     """Representation of an audio sample to be used with Google Speech API.
 
-    :type content: bytes
-    :param content: (Optional) Byte stream of audio.
+    :type content: bytes or file
```

Comment:
Is there a reason to allow `content` to be two different types? Is it a large drag on users to just have two different input params?

@tseaver @jonparrott WDYT?